modules/assigns/assignExists.js

```diff
@@ -0,0 +1,19 @@
+const models = require('../../loading/loading')
+const Promise = require('bluebird')
+
+module.exports = Promise.method(function assignExists (assignAttributes) {
+  return models.Assign
+    .findOne({
+      where: {
+        userId: assignAttributes.userId,
+        TaskId: assignAttributes.taskId
+      }
+    }).then(assign => {
+      if (!assign) return false
+      return assign
+    }).catch(error => {
```

Comment:
With `promise` method we don't need try catch here, it throws a reject promise and we can handle this error on the controller side